.github/workflows/nodejs.yml

```diff
@@ -25,3 +22,68 @@ jobs:
         yarn test
       env:
         CI: true
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+      with:
+        ref: fix/docker-build
```

Comment:
Does this set it up to only run on this branch? As in the comment before, only on pushing to develop & master